drivers/logserial.py

```diff
@@ -14,57 +14,47 @@
 
 
 import serial
-from threading import Thread, Event
+from threading import Thread
 
 from lib.logger import LogWriter
 from drivers.bus import BusShutdownException
 
 
-class LogSerial(Thread):
-    def __init__(self, config, bus, com=None):
-        Thread.__init__(self)
-        self.setDaemon(True)
-        self.should_run = Event()
-        self.should_run.set()
+class LogSerial:
+    def __init__(self, config, bus):
+        self.input_thread = Thread(target=self.run_input, daemon=True)
+        self.output_thread = Thread(target=self.run_output, daemon=True)
 
-        if com is None:
-            if 'port' in config:
-                self.com = serial.Serial(config['port'], config['speed'])
-                self.com.timeout = 0.01  # expects updates < 100Hz
+        if 'port' in config:
+            if config.get('rtscts'):
+                self.com = serial.Serial(config['port'], config['speed'], rtscts=True)
+                self.com.setRTS()
```

Comment:
I realize this is probably tested with hw, but I am still wondering. The [rtscts](http://pyserial.readthedocs.io/en/latest/pyserial_api.html#serial.Serial.rtscts) enables flow control, however [rts](http://pyserial.readthedocs.io/en/latest/pyserial_api.html#serial.Serial.rts) toggles the pin directly. Why are we doing it?

Reply:
For communication with CAN bridge. I would review it in #48 if necessary.